LOCALIZE.md

```diff
@@ -93,7 +93,7 @@ For each `word`, an optional _.png_ image exists. In Swahili, the words _mbu, ki
 
 ### 4. Stories
 Localised stories each have an `id` and reside in `assets/oc-reading/books/xr-[id]/`. Each story has configuration file `book.xml`.
-Presentational apsects of the story are defined on the `book` element. Each `page` contains one or more localised `para`. A `page` can have an optional `picjustify` attribute to specify the page layout.
+Presentational aspects of the story are defined on the `book` element. Each `page` contains one or more localised `para`. A `page` can have an optional `picjustify` attribute to specify the page layout.
 
 In lower level stories, there is a syllable breakdown for each word, delimited by `/`. The title and first page of the _A very tall man_ story in Swahili are represented like this:
 
```

Comment:
Can you add the english version of the example?

Reply:
Updated